src/Module/BaseProfile.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Friendica\Module;
+
+use Friendica\App;
+use Friendica\BaseModule;
+use Friendica\Core\Hook;
+use Friendica\Core\Renderer;
+use Friendica\DI;
+
+class BaseProfile extends BaseModule
+{
+	/**
```

Comment:
Maybe you can add a short summary about the method :) a sentence would be enough I guess ^^

Reply:
It's going to be so redundant with the name of the method you will be twice sorry you asked. 😛